.github/ISSUE_TEMPLATE/5.priority-support.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Priority support: SLA ⏰'
```

Comment:
Are we going to add the same template to the https://github.com/mui/material-ui?